.github/workflows/add-pr-approval-label.yml

```diff
@@ -17,7 +17,6 @@ jobs:
       - uses: ./
         with:
           helper: add-pr-approval-label
-          teams: Repo-Maintainers
```

Comment:
We should accept code owners path as input since CODEOWNERS file could be in multiple places. https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#codeowners-file-location

Reply:
The codowners-utils package should handle all supported CODEOWNERS locations.